cpp/oneapi/dal/algo/pca/test/fixture.hpp

```diff
@@ -141,10 +196,42 @@ class pca_test : public te::crtp_algo_fixture<TestType, Derived> {
         check_variances(bs, variances);
     }
 
+    void check_train_result_online(const pca::descriptor<Float, Method>& desc,
+                                   const te::dataframe& data,
+                                   const pca::train_result<>& result) {
+        const auto [eigenvalues, eigenvectors] = unpack_online_result(result);
+
+        check_online_shapes(desc, data, result);
+        check_nans(result);
+
+        INFO("check if eigenvectors order is descending")
+        this->check_eigenvalues_order(eigenvalues);
+
+        INFO("check if eigenvectors matrix is orthogonal")
+        check_eigenvectors_orthogonality(eigenvectors);
+    }
+
     void check_infer_result(const pca::descriptor<Float, Method>& desc,
```

Comment:
why is this function empty? shouldnt it be validating the results? or does that happen elsewhere

Reply:
We dont have accuracy check for results in batch. Will be added in separate pr

Reply:
would it ever fail?